src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpOntologyDAO.scala

```diff
@@ -59,4 +59,10 @@ class HttpOntologyDAO(implicit val system: ActorSystem, implicit val executionCo
     implicit val timeout:Timeout = 60.seconds // timeout to get the host connector reference
     for (Http.HostConnectorInfo(connector, _) <- IO(Http) ? ontologyHostSetup) yield connector
   }
+
+  override def status: Future[SubsystemStatus] = {
+    val ontologyStatus = unAuthedRequestToObject[Map[String, DropwizardHealth]](Get(ontologyUri.withPath(Uri.Path("/status"))))
```

Comment:
I'm not convinced this actually works. Before I left for vacation, I was on the verge of changing it to a `adminAuthedRequestToObject` to solve a 403/forbidden issue.

I will spin up the whole system locally and determine whether the above change is necessary.

(As I understand it, it's not possible to do a `authedRequestToObject` from `/status` because there is no user involved, per Andrea.)